torch/nn/functional.py

```diff
@@ -4638,6 +4638,12 @@ def interpolate(  # noqa: F811
             "Anti-alias option is restricted to bilinear and bicubic modes and requires a 4-D tensor as input"
         )
 
+    if not torch.jit.is_scripting() and scale_factors is not None:
+        # We need to specialize symfloats for now. Eventually we should do a tensorify pass in dynamo.
```

Comment:
This is a suspicious spot to do it, would have expected later

Reply:
Did you have a place in mind? Generally for all of these fixes I'm specializing right before the point it needs to in the stacktrace (via debugger + jumping up and down stacks). If there's a better method to find the precise place to put these specializations, do let me know!